overlord/snapshotstate/snapshotmgr.go

```diff
@@ -58,6 +60,7 @@ type SnapshotManager struct {
 	state *state.State
 
 	lastForgetExpiredSnapshotTime time.Time
+	cleanupAbandondedImportsRan   bool
```

Comment:
Not needed anymore after moving to StartUp?

Reply:
Nice catch!